pkg/dab/find.go

```diff
@@ -0,0 +1,237 @@
+package dab
+
+import (
+	"bufio"
+	"errors"
+	"io"
+	"io/fs"
+	"path/filepath"
+
+	"github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/codec/json"
+	"github.com/serum-errors/go-serum"
+	"github.com/warptools/warpforge/wfapi"
+)
+
+// ActionableSearch is a bitfield for what the ActionableFromFS function should treat as acceptable findings.
+type ActionableSearch uint8
+
+const (
+	ActionableSearch_None    ActionableSearch = 0
```

Comment:
@warpfork and I discussed this very briefly and it may be more readable being explicit, but I still think it's fine to use ` = 1 << iota -1` here and let the compiler handle the flag setting by declaration order.